pkgs/development/python-modules/notmuch2/default.nix

```diff
@@ -20,17 +20,20 @@ buildPythonPackage {
   ];
 
   # since notmuch 0.35, this package expects _notmuch_config.py that is
-  # generated by notmuch's configure script
+  # generated by notmuch's configure script. Substitute our nix-crafted one.
   postPatch = ''
-    cp ${notmuch.bindingconfig}/_notmuch_config.py .
+    cat > _notmuch_config.py <<EOF
+    NOTMUCH_VERSION_FILE='${notmuch.bindingconfig}/version.txt'
+    NOTMUCH_INCLUDE_DIR='${notmuch.out}/include'
+    NOTMUCH_LIB_DIR='${notmuch.out}/lib'
+    EOF
   '';
 
   # no tests
   doCheck = false;
   pythonImportsCheck = [ "notmuch2" ];
 
   meta = with lib; {
-    broken = stdenv.isDarwin;
```

Comment:
Is this unrelated, or was it only broken on darwin?

Reply:
AFAICT it was broken only on darwin. I suspect sourceRoot is handled slightly differently there, and that's why version.txt happened to be accessible on linux? I'm not sure really